dali/python/nvidia/dali/ops.py

```diff
@@ -24,6 +24,7 @@
 def _docstring_generator(cls):
     __cpu_ops = set(b.RegisteredCPUOps())
     __cpu_ops.add("TFRecordReader")
+    __cpu_ops.add("PythonFunction")
```

Comment:
was this necessary? I thought that the registered ops are automatically updated after loading the library

Reply:
Python wrappers are not registered ops. These names need to be manually added to the list only for documentation generation reasons.